var/spack/repos/builtin/packages/vim/package.py

```diff
@@ -50,9 +50,18 @@ class Vim(Package):
     variant('ruby', default=False, description="build with Ruby")
     depends_on('ruby', when='+ruby')
 
+    variant('lua', default=False, description="build with Lua")
+    depends_on('lua', when='+lua')
+
+    variant('perl', default=False, description="build with Perl")
+    depends_on('perl', when='+perl')
+
     variant('cscope', default=False, description="build with cscope support")
     depends_on('cscope', when='+cscope', type='run')
 
+    variant('failmissing', default=True,
```

Comment:
Correct, I wasn't sure whether it is okay to just use it, without having a way of disabling it.